app/uk/gov/hmrc/brm/models/brm/HttpResponse.scala

```diff
@@ -57,78 +58,78 @@ trait HttpResponse {
 
 object CustomErrorResponse extends HttpResponse {
```

Comment:
@chris-ianson We can remove this duplicate object and just call ErrorResponses.handle directly.